### ISSUE TITLE
include-file paths not expanded

### ISSUE THREAD

**Issue by rosshadden** [jaagr/polybar#603]
Paths for `include-file` do not expand the `~` prefix, which would be very helpful because some of us have multiple computers with different usernames.  Some of the user-contributed modules use `~` in paths for `exec`, `click-left`, etc., so it's strange that it works in some places and not others.

Also variables cannot be used in include paths.  Is this a feasible change?  I didn't want to open up a separate issue for something so closely-related.

**Comment by jaagr** [comment]
Yeah it's feasable change. I even thought I fixed it already but I'll have a look.

**Comment by Mange** [comment]
I was trying something like this:

```
[bar/base]
inherit = bar/core
include-file = ${env:POLYBAR_SCREEN_CONFIG_FILE:~/.config/polybar/default_screen.ini}
```

but I get this error:

```
error: Invalid include file "${env:POLYBAR_SCREEN_CONFIG_FILE:~/.config/polybar/default_screen.ini}" defined on line 39
```

I suspect that is what @rosshadden meant by "variables cannot be used in include paths".

---

In order to avoid [The XY Problem](http://xyproblem.info/): I was trying to do this because I want a portable set of dotfiles. I need them to work at my workstation at home, my laptop at work, and my laptop at work when plugged into external screen(s).
Some screens are HiDPI and I need to have bigger sizes to be able to read text on the bar on those screens.

I'm gonna extract several env variables to inject instead now, but this looked like a nice way to just have different files for different configurations.

**Comment by NBonaparte** [comment]
What I thought he meant was something like `include-file  = $XDG_CONFIG_HOME/polybar/file`, which should be fixed by #724.

**Comment by Mange** [comment]
Yes, that makes sense.
